cmd/kyverno/main.go

```diff
@@ -102,6 +103,7 @@ func main() {
 	flag.BoolVar(&autoUpdateWebhooks, "autoUpdateWebhooks", true, "Set this flag to 'false' to disable auto-configuration of the webhook.")
 	flag.Float64Var(&clientRateLimitQPS, "clientRateLimitQPS", 0, "Configure the maximum QPS to the master from Kyverno. Uses the client default if zero.")
 	flag.IntVar(&clientRateLimitBurst, "clientRateLimitBurst", 0, "Configure the maximum burst for throttle. Uses the client default if zero.")
+	flag.BoolVar(&autogenInternals, "autogenInternals", false, "Enables autogen internal policies. When this is 'true' policy rules should not be mutated.")
```

Comment:
Do we need to note it down in the Changelog? Or doc update https://kyverno.io/docs/installation/#container-flags?

Reply:
Yeah, good catch, I'll add that in the changelog in the next PR.

Reply:
Fixed.

Reply:
Can you point me to the PR? I cannot find it in current `main`.

Reply:
It will be this one https://github.com/kyverno/kyverno/pull/3410

Reply:
Didn't add the in the `CHANGELOG` yet though, but I will.

Reply:
Great, thanks!